lib/stylelint/10.0.x/README.md

```diff
@@ -0,0 +1,20 @@
+<h1 align="center">
+   code style - stylelint 10.0.x
+</h1>
+
+## About
+
+This sub-package hosts the [stylelint](https://stylelint.io) configuration for stylelint 10.0.x.
+
+## Usage
+
+Adapt the content of your `.stylelintrc` file as follows:
+
+```json
```

Comment:
This needs to be type `text`, otherwise the comment `// your rules` appears highlighted in red :(

The same for the other READMEs